content/en/apps/features/supervision/_index.md

```diff
@@ -46,3 +46,12 @@ The CHT can be configured to create [Tasks]({{< relref "apps/features/tasks" >}}
 ### Deployment Case Study
 [Muso](https://www.musohealth.org), a leading community health organization and major contributor to the CHT, has implemented ”360º supervision”, achieving some of the lowest child mortality rates in sub-Saharan Africa. This model provides dedicated mentorship and supportive supervision to CHWs tailored to each CHW's particular strengths and challenges. A key theme of our [human-centered approach](https://medicmobile.org/design) was the idea of using data to improve one-to-one supervision, rather than using analytics to replace Supervisors. Read more about findings from a recent [randomized controlled trial](https://medicmobile.org/blog/new-study-precision-supervision-and-personalized-feedback-dashboards-improve-chw-performance-in-mali).
 {{% /pageinfo %}}
+
+## Supervisor CHW create
+This feature enables a supervisor to create a new user under an existing place that has been selected. Supervisor CHW Create is compatible with CHTs offline first features. When offline, the supervisor can create the user and a login token provided in the outgoing message tab.While users will not exist until the supervisor synchronizes, and the newly created user needs connectivity to access their login token and log in the first time, all other process can be completed offline and be synchronized at a later time.
+
+To [configure]({{< ref "apps/reference/app-settings/transitions#create_user_for_contacts" >}}) this feature, one needs to make some changes to the app_settings file. These changes include enabling contacts to create users,enabling token login and setting the app url to the url of the development instance.
```

Comment:
I personally think this is more technical than this page needs to get. Consider just adding the config link  under "Related Content"?